CHANGELOG.md

```diff
@@ -2,6 +2,41 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+# 10.0.0-beta.17 - April 12, 2021
+## Breaking changes ⚠️
+* [Annotation plugin] Rename annotation classes ([#227](https://github.com/mapbox/mapbox-maps-android/pull/227))
```

Comment:
Please also add how exactly classes were renamed - we were already asked about it several times.